ui/src/app/components/routing.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+import {withRoute} from 'app/routes';
+import {history} from 'app/services/history';
+
```

Comment:
Possibly another component we'd want eventually here would be something like a `<RouterLink to="/" />` which renders an `<a href="/">...</a>` with a click handler that handles this via the html lib.

Not something I'm necessarily asking you to add to the PR but noting we may need it and raising in case you had any thoughts in this area.

Reply:
Yes, that's definitely an abstraction we'll want.